modules/trimmomatic/trimmomatic.nf

```diff
@@ -0,0 +1,79 @@
+nextflow.enable.dsl = 2
+
+
+params.save_mode = 'copy'
+params.results_dir = "${params.outdir}/trimmomatic"
+params.should_publish = true
+
+
+process TRIMMOMATIC {
+    tag "${genomeName}"
+    publishDir params.results_dir, mode: params.save_mode, enabled: params.should_publish
+
+    input:
+    tuple val(genomeName), path(genomeReads)
+
+    output:
+    tuple val(genomeName), path("*_{R1,R2}.p.fastq.gz")
+
+    script:
+
+    fq_1_paired = genomeName + '_R1.p.fastq.gz'
+    fq_1_unpaired = genomeName + '_R1.s.fastq.gz'
+    fq_2_paired = genomeName + '_R2.p.fastq.gz'
+    fq_2_unpaired = genomeName + '_R2.s.fastq.gz'
+
+    def adapterFile = "/usr/local/share/trimmomatic-0.35-6/adapters/NexteraPE-PE.fa"
```

Comment:
When you change the `trimmomatic` version, this would also need to be updated.

If possible,  could you please look into some NF tricks to make sure that this `adapterFile` is located correctly or all versions of `trimmomatic` which a user might use.

Perhaps, the ability to specify a trimming file `adapterFile`  is also worth exposing to the user - what do you think? 🤔

Reply:
yes! I'll follow the pattern of wgslineage1 project.

>If possible, could you please look into some NF tricks to make sure that this adapterFile is located correctly or all versions of trimmomatic which a user might use.

What do you think about having a pr just to discuss this? I think that this problem will require more after trimmomatic is fully working

Reply:
> What do you think about having a pr just to discuss this? I think that this problem will require more after trimmomatic is fully working

Agreed, please freel free to initiate another PR only for managing only the `adapterFiles`.